src/Sylius/Bundle/AddressingBundle/Repository/ZoneRepository.php

```diff
@@ -80,7 +80,9 @@ public function createByAddressQueryBuilder(AddressInterface $address, ?string $
             $queryBuilder->setParameter('provinceCode', $address->getProvinceCode());
         }
 
-        $queryBuilder->andWhere($queryBuilder->expr()->orX(...$orConditions));
+        if ($orConditions !== []) {
```

Comment:
```suggestion
        if (!empty($orConditions)) {
```

Reply:
I really don't like using `empty` function 😄 

https://dev.to/klnjmm/never-use-empty-function-in-php-4pb0